app/assets/stylesheets/styles/post-list.scss

```diff
@@ -0,0 +1,26 @@
+#list-container {
+  display: grid;
+  grid-template-columns: 25% 50% 25%;
+}
+
+#list-middle-container {
+  grid-column-start: 2;
+  grid-column-end: 3;
+}
+
+#list-header-container {
+  border-bottom: 1px solid black;
+}
+
+#post-list {
+  margin-top: 3px;
+}
+
+h1 {
+  display: inline-block;
+}
+
+#create-post-link {
+  float: right;
+  margin-top: 24px;
+}
```

Comment:
This file is too ID happy.  Use classes.  Also you are missing a newline at the end of the file

Reply:
The element was given the class createPostLink as well in other elements in post-list-scss